mlrun/features.py

```diff
@@ -164,7 +284,112 @@ def check(self, value):
         return ok, args
 
 
+class MinMaxLenValidator(Validator):
+    """Validate min/max length value ranges"""
+
+    kind = "minmaxlen"
+    _dict_fields = Validator._dict_fields + ["min", "max"]
+
+    def __init__(
+        self, check_type: bool = None, severity: str = None, min=None, max=None
+    ):
+        """Validate min/max length value ranges
+
+        example::
+
+            from mlrun.features import MinMaxLenValidator
+
+            # Add length validator to the feature 'ticker', where valid
+            # minimal length is 1 and maximal length is 10
+            quotes_set["ticker"].validator = MinMaxLenValidator(
+                min=1,
+                max=10,
+                severity="info"
+            )
+
+        :param check_type:  check feature type e.g. True, False
+        :param severity:    severity name e.g. info, warning, etc.
+        :param min:         minimal valid length size
+        :param max:         maximal valid length size
+        """
+        super().__init__(check_type, severity)
+        self.min = min
+        self.max = max
+
+    def check(self, value):
+        ok, args = super().check(value)
+        if ok:
+            if self.min is not None:
+                if len(value) < self.min:
```

Comment:
Do we need to consider the possibility that `len(value)` will raise an exception? e.g. `len(5)`

Reply:
It makes sense to catch exception, typically e.g. TypeError: object of type 'bool' has no len(). I will do update.